react/admin/ReturnDetails/components/VerifyItems/ProductActionStatus.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { IconSuccess, IconFailure, IconWarning } from 'vtex.styleguide'
+import { FormattedMessage } from 'react-intl'
+
+type ActionStatus = 'deny' | 'approve' | 'partially-approve'
+
+const getActionStatus = ({
+  quantity,
+  selectedQuantity,
+}: {
+  quantity: number
+  selectedQuantity: number
+}): ActionStatus =>
+  selectedQuantity === 0
+    ? 'deny'
+    : selectedQuantity === quantity
+    ? 'approve'
+    : 'partially-approve'
+
+export const ProductActionStatus = ({
+  quantity,
+  selectedQuantity,
+}: {
+  quantity: number
+  selectedQuantity: number
+}) => {
+  const actionStatus = getActionStatus({ quantity, selectedQuantity })
+
+  return (
+    <>
+      {actionStatus !== 'approve' ? null : (
```

Comment:
A nitpick, but same thing can be achieved with just &&
`actionStatus !== 'approve' && (<Component>)`

Reply:
I stopped using `&&` because of this: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx